pkg/useragent/useragent_test.go

```diff
@@ -0,0 +1,20 @@
+package useragent
+
+import (
+	"runtime"
+	"testing"
+)
+
+func TestUnameNotEmpty(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		u := getUname()
+
+		if u == "" {
+			t.Error("Failed to pull unix name")
```

Comment:
We use the `require` library in our tests to make it easier to test various assertions. Can you change this to:
```go
require.NotEmpty(t, u);
```

Reply:
done